Dockerfile.main

```diff
@@ -1,41 +1,37 @@
-FROM ubuntu:xenial
```

Comment:
I've noticed `Dockerfile.main` and `Dockerfile.test` are pretty much the same. The diff is:
```
3c3
< LABEL description="Lisk Docker Image - Mainnet" version="1.4.2"
---
> LABEL description="Lisk Docker Image - Testnet" version="1.4.2"
30c30
< RUN wget https://downloads.lisk.io/lisk/main/lisk-source.tar.gz -O lisk-source.tar.gz
---
> RUN wget https://downloads.lisk.io/lisk/test/lisk-source.tar.gz -O lisk-source.tar.gz
61,62c61,62
< EXPOSE 8000
< ENTRYPOINT ["./setup.sh", "main"]
---
> EXPOSE 7000
> ENTRYPOINT ["./setup.sh", "test"]
```
I reckon we can create a unique file and pas the option `{test|main}`. Also we might have a third option `local` with more differences.